scripts/bootstrap.js

```diff
@@ -36,7 +36,8 @@ function authorize() {
         });
 }
 
-function getOrCreateCollection(collection, token) {
+function getOrCreateCollection(collection, token, attributes) {
+    attributes = attributes || {};
```

Comment:
As someone recently showed me... ES6 allows default parameters, so line 40 can be consolidated into 39.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters